### DISCUSSION TITLE
Is there a setting or config to change color of the text in vi editor, not terminal?

### DISCUSSION THREAD

**Discussion by lifeoncloud** [Eugeny/tabby#8614] [Q&A]
Hello,
First of all, I'm really appreciated with your work.
I have been enthusiastically introducing Tabby to engineers around me since I found out about it.
I think Tabby is really awesome.

I was wondering if it is possible to change the color of the text inside the vi editor.
I want to custom the color of the colon in vi editor darker color in a light theme.

I checked in the link below that the 'Appearance' menu only defines the color of the Terminal.
https://github.com/Eugeny/tabby/discussions/5258 

Is there a setting or config.yaml to custom color of the colon in vi editor?
![vi](https://github.com/Eugeny/tabby/assets/62153151/dd4d0289-1703-4115-a770-0b7a7fa132a4)
<img width="731" alt="which-color" src="https://github.com/Eugeny/tabby/assets/62153151/f73b617e-bc15-4e21-8444-aac13a3e6c83">

**Comment by Eugeny** [comment]
Hi, as mentioned https://github.com/Eugeny/tabby/discussions/5258, a terminal can't make a program's black-and-white output colored. You need to get vim to do syntax highlighting first https://linuxhandbook.com/syntax-highlight-vim/
